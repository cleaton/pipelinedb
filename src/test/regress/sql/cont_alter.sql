CREATE CONTINUOUS VIEW test_cont_alter AS SELECT x::int FROM stream;

ALTER VIEW test_cont_alter RENAME TO lolcat;
ALTER TABLE test_cont_alter_mrel RENAME TO lolcat;
ALTER VIEW test_cont_alter ALTER COLUMN x SET DEFAULT 10;
ALTER TABLE test_cont_alter_mrel DROP COLUMN x;

DROP CONTINUOUS VIEW test_cont_alter;
