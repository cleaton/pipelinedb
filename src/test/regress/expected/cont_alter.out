CREATE CONTINUOUS VIEW test_cont_alter AS SELECT x::int FROM stream;
ALTER VIEW test_cont_alter RENAME TO lolcat;
ERROR:  "test_cont_alter" is not a view
ALTER TABLE test_cont_alter_mrel RENAME TO lolcat;
ERROR:  cannot rename materialization table "test_cont_alter_mrel" for continuous view "test_cont_alter"
ALTER VIEW test_cont_alter ALTER COLUMN x SET DEFAULT 10;
ERROR:  "test_cont_alter" is not a view
ALTER TABLE test_cont_alter_mrel DROP COLUMN x;
ERROR:  cannot alter materialization table "test_cont_alter_mrel" for continuous view "test_cont_alter"
DROP CONTINUOUS VIEW test_cont_alter;
